metapool/prep.py

```diff
@@ -665,22 +666,25 @@ def preparations_for_run_mapping_file(run_path, mapping_file):
     return output
 
 
-def parse_prep(prep_path):
-    """Parses a prep file into a DataFrame
+def parse_prep(prep_path, numeric_index=False):
```

Comment:
This new parameter seems to only be used in tests, if that's the case why not change the tests so this parameter is not needed?